data_transfer/lib/vttsma.py

```diff
@@ -0,0 +1,90 @@
+# NOTE: some methods exist for convience and will be advanced
+# for the CVS, e.g. serial_by_device_uuid might to hit a live 
+# endpoint rather than pull from a CSV file.
+
+from data_transfer.config import config
+from data_transfer.services import dmpy
+from pathlib import Path
+from dataclasses import dataclass
+from typing import Optional
+from functools import lru_cache  # note: @cache exists in py >= 3.9
+import csv
```

Comment:
I don't think we are using this anymore.. are we?

Reply:
We do not -- it's been moved to `utils/__init__.py`